bin/node/cli/benches/block_production.rs

```diff
@@ -52,6 +54,7 @@ fn new_node(tokio_handle: Handle) -> node_cli::service::NewFullBase {
 		Sr25519Keyring::Alice.to_seed(),
 		"network/test/0.1",
 		Default::default(),
+		PreSharedKeyConfig { pre_shared_key: PreSharedKeySecret::File(PathBuf::from("./pre_shared_key")) },
```

Comment:
have you tried to run those benchmark tests? Does it work? As I understand we need to create this file first to make it run - if so, we should create this test file here and commit it

Reply:
ok